### PR TITLE
Add a new system for rating HW on its learning goals

### PR DESCRIPTION
**Pull request by tildeee**
**I generally just... made stuff up**, but I figured it was easier to write something down and talk about it, so let me know what you think.

Would you be willing to add some combination of [R], [S], or [N] on each reading assignment or live code viewing?

### CODE REVIEW

File: hw-learning-goal-label-key.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# HW Learning Goal Label Key
+This document describes the labels we give some of our homework assignments. These labels should help a student understand what the learning goal of the assignment.
+
+Often, we will assign homework in the following or similar structure:
+
+```
+3. Read about writing pseudocode: https://github.com/Ada-Developers-Academy/ [R]
+```
```
Comment (droberts-sea) [inline hw-learning-goal-label-key.md]: Might be more readable to have the rating before the link

File: hw-learning-goal-label-key.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+An assignment with the Reinforcement [R] label indicates that the goal of the reading is to go over a concept that has already been introduced into the curriculum.
+
+## Supplementing, Adding, or Extending Context [S]
+
+When an assignment has this label, it serves to add additional context to familiar concepts. Materials and assignments with this label will likely increase and round-out your understanding of a topic. It gives either a different way of looking at a concept, explains a concept more holistically, or shares a way that the concept is applied in a practical way.
+
+## New Material and Immediately Relevant [N]
+
+Reading with this label usually indicates that it is introducing a new concept and new material. If we are assigning new material as reading homework, it will become immediately relevant, likely the very next day.
+
```
Comment (droberts-sea) [inline hw-learning-goal-label-key.md:33]: These are the ones we consider to be mandatory, right? Can we make that explicit?

Comment (droberts-sea) [reply hw-learning-goal-label-key.md:33]: >These are the materials we consider to be the most important to our curriculum. If you read nothing else, read the [N]s.

**Review by droberts-sea — CHANGES_REQUESTED** [review]

**Review by CheezItMan — APPROVED** [review]
I pretty much agree with Dan's comments, otherwise looks good.

**Comment by tildeee** [comment]
@droberts-ada just pushed an update re: your suggestions!

**Review by droberts-sea — APPROVED** [review]

**Pull request merged**